editors/MacVim/Portfile

```diff
@@ -77,7 +77,7 @@ build.args          XCODEFLAGS="CACHE_ROOT=${workpath}/caches"
 
 # this port does not build with the new xcode build system at present
 if {[vercmp ${xcodeversion} 10.0] >= 0} {
-    build.args    XCODEFLAGS="CACHE_ROOT=${workpath}/caches -UseNewBuildSystem=NO"
+    build.args    XCODEFLAGS="CACHE_ROOT=${workpath}/caches -UseNewBuildSystem=NO MACOSX_DEPLOYMENT_TARGET=${macosx_deployment_target}"
```

Comment:
The notion looks correct, but please add this to a separate `build.args-append` line. This has nothing to do with what is described in the comment above this block.

Reply:
But it needs to be appended to the XCODEFLAGS variable within build.args, not as a new string added to the end of build.args. I suppose we could build up the XCODEFLAGS content on multiple lines, and then set the build.args once...